src/apps/pika-protocol-v3/optimism/pika-protocol-v3.balance-fetcher.ts

```diff
@@ -47,7 +54,7 @@ export class OptimismPikaProtocolV3BalanceFetcher implements BalanceFetcher {
         const rewardToken = contractPosition.tokens.find(isClaimable)!;
 
         const [stakedBalanceRaw, rewardBalanceRaw] = await Promise.all([
-          multicall.wrap(contract).getShare(address),
+          this.getBalanceInVault(contract, address),
```

Comment:
You're losing the `multicall.wrap` here, so pass in the multicall instance to `this.getBalanceInVault`. Multicall is important so we can batch calls together and keep costs down.